src/test/java/seedu/todo/logic/commands/CompleteCommandTest.java

```diff
@@ -28,8 +29,12 @@ public void testMarkComplete() throws Exception {
         setParameter("3");
         execute(true);
         ImmutableTask markedComplete = getTaskAt(3);
+        assertEquals(result.getFeedback(), 
```

Comment:
I don't think this is a good test. It's basically a copy of the code it's testing. Try checking for something else in the output.